Cauldron/Assets/Scripts/UI/RecipeBookButton.cs

```diff
@@ -54,5 +100,11 @@ public void OnPointerExit(PointerEventData eventData)
                 transf.DOPlay();
             }
         }
+        
+        private void OnDestroy()
+        {
+            book.AvailableChange -= gameObject.SetActive;            
```

Comment:
If you don't have a subscription, you don't need to unsubscribe)
If there is a subscription somewhere, looks like this SetActive should be placed in ChangeBookAvailable.